bindings/java/Cargo.toml

```diff
@@ -32,10 +32,13 @@ crate-type = ["cdylib"]
 doc = false
 
 [dependencies]
-jni = "0.21.1"
 opendal.workspace = true
 
+enum-iterator = { version = "1.4.1", optional = true }
+jni = "0.21.1"
 once_cell = "1.17.1"
-tokio = { version = "1", features = ["full"] }
-
+tokio = { version = "1.28.1", features = ["full"] }
 num_cpus = "1.15.0"
+
+[features]
```

Comment:
We don't need to add a new feature for tests, we can move this into `dev-dependencies`

Reply:
I'll try it. But it's about we load the compiled dylib, so I don't know how to build a dylib with `dev-dependencies` bundled.

Reply:
Maybe we don't need to add `into_ordinal` for `ErrorKind`.